platform_versioned_docs/version-24.1/enterprise/kubernetes-minimal-install.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: Install Seqera Platform on Kubernetes
+description: Deploy Seqera Platform Enterprise with Kubernetes
+date: "11 Sep 2024"
+tags: [kubernetes, deployment]
+---
+
+This installation guide describes how to install Seqera Platform on a Kubernetes cluster.
+
```

Comment:
Potentially expand on this and explain what the outcomes will be. 

This installation guide describes how to install Seqera Platform on a Kubernetes cluster, at the end of the guide you will have a version of the platform running be able to log in etc.